app/src/main/java/de/tum/in/tumcampusapp/component/ui/tufilm/KinoDao.java

```diff
@@ -29,7 +29,7 @@
     @Query("SELECT id FROM kino ORDER BY id DESC LIMIT 1")
     Maybe<String> getLastId();
 
-    @Query("SELECT * FROM kino ORDER BY id LIMIT 1 OFFSET :position")
+    @Query("SELECT * FROM kino LIMIT 1 OFFSET :position")
```

Comment:
`LIMIT` without `ORDER BY` doesn't specify which element it returns.
Maybe we want to `ORDER BY date` instead

Reply:
@pfent Yes, ordering by date should work for now. And I finally get why ordering by id does not work (although the ids are in the right order, we get them from the api). The problem is that the ids are strings, so the Jumanji movie had id '100' which is smaller than '99' etc. 😣